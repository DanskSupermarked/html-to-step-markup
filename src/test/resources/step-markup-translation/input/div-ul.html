<div class="standardArticle_1">Havemøbelsæt bestående af 1 bord samt 6 stole.<br /> Bordet er et udtræksbord og er 100 x 206 cm og udslået 100 x 275 cm.<br /> Bordet har et stel i sort pulverlakeret aluminium og en bordplade i sort aintwood.<br /> Brava stolene har justerbar ryg og er med aluminiumsstel og flet i sort polyrattan.<br /> Brava sættet passer perfekt til den store familie, eller dem der bare foretrækker masser af bordplads.<br /> Materialet aintwood er nemt at vedligeholde, så hvis man gerne vil have et minimum af vedligeholdelse, så er dette sæt det helt rigtige.<br /> </div><ul><li>Brava havesæt med udtræksbord i aintwood</li><li>6 stole med justerbar ryg i sort polyrattan</li><li>Stort udtræksbord på 100 x 206/275 cm</li><li>Minimum af vedligeholdelse</li></ul><div class="standardArticle_1"><br /> Aintwood: Hvis du elsker træ, men ikke vil bruge tid på vedligeholdelse så er aintwood noget for dig. Det er et plastmateriale der ligner træ både ved berøring og i udtryk. Materialet fås i et væld af farver. Aintwood er stort set vedligeholdelsesfrit og skal blot rengøres med vand og sæbe.<br /> <br /><br /> Polyrattan:Det er et plastbaseret materiale, der betragtes som fremtiden inden for havemøbelproduktion. Materialet er UV-bestandigt og modstandsdygtigt overfor naturens luner. Rengøres let med vand og sæbe.<br /> <br /><br /> Brava sæt med 6 stole er altså et ideelt havemøbelsæt til dig, der foretrækker et minimum af vedligeholdelse af dine havemøbler og plads til hele familien.</div>
