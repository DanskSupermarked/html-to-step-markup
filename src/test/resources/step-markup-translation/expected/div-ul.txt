Havemøbelsæt bestående af 1 bord samt 6 stole.
Bordet er et udtræksbord og er 100 x 206 cm og udslået 100 x 275 cm.
Bordet har et stel i sort pulverlakeret aluminium og en bordplade i sort aintwood.
Brava stolene har justerbar ryg og er med aluminiumsstel og flet i sort polyrattan.
Brava sættet passer perfekt til den store familie, eller dem der bare foretrækker masser af bordplads.
Materialet aintwood er nemt at vedligeholde, så hvis man gerne vil have et minimum af vedligeholdelse, så er dette sæt det helt rigtige.
<bulletlist>
<bullet>Brava havesæt med udtræksbord i aintwood</bullet>
<bullet>6 stole med justerbar ryg i sort polyrattan</bullet>
<bullet>Stort udtræksbord på 100 x 206/275 cm</bullet>
<bullet>Minimum af vedligeholdelse</bullet>
</bulletlist>

Aintwood: Hvis du elsker træ, men ikke vil bruge tid på vedligeholdelse så er aintwood noget for dig. Det er et plastmateriale der ligner træ både ved berøring og i udtryk. Materialet fås i et væld af farver. Aintwood er stort set vedligeholdelsesfrit og skal blot rengøres med vand og sæbe.

Polyrattan:Det er et plastbaseret materiale, der betragtes som fremtiden inden for havemøbelproduktion. Materialet er UV-bestandigt og modstandsdygtigt overfor naturens luner. Rengøres let med vand og sæbe.

Brava sæt med 6 stole er altså et ideelt havemøbelsæt til dig, der foretrækker et minimum af vedligeholdelse af dine havemøbler og plads til hele familien.
